docs/README.md

```diff
@@ -3,8 +3,10 @@
     - [Install **source{d} CE**](./quickstart/2-install-sourced.md)
     - [Run **source{d} CE**](./quickstart/3-init-sourced.md)
     - [Explore Your Data](./quickstart/4-explore-sourced.md)
+    - [Next steps](./usage/README.md)
```

Comment:
Aren't we spreading the menus around too much? `usage/README` contains the same list of links as the section right below this one in this doc. But reading this menu file only, it looks like we have 'Quick/Next steps` and `Usage`, when they are the same contents.

Reply:
(explained by the **TL;DR** and by https://github.com/src-d/sourced-ce/issues/126#issuecomment-507810661)

If the users visit our quickstart, and keep clicking "next>" they will follow a clear path from _"requirements"_, to _"next topics that you should know"_ (with links to those topics).

Again: it's difficult to satisfy both user experiences: GH and GitBook, with its current limitations, and to satisfy all our requirements about having a clear and brief entry point for our docs (`README.md`).

Imo, this PR provides a good good enough UX on GitHub, and does not difficult the experience in GitBook because there, all pages are highlighted in the aside, so the users always know where they are, and what's next.

Reply:
A possible improvement to make the `docs/usage/README.md` richer than the aside summary:
>To explain the content of each link, so the user would know more about it, compared with what it currently appears in the aside, e.g.
>- In the aside: "_Babelfish UAST_"
>- In `docs/usage/README.md`: "_[Babelfish UAST](./bblfsh.md), how to extract code features and understand code structure in a language-agnostic way_"

I don't feel it's really urgent, so if approved, it could be done in a separate PR.